io.c

```diff
@@ -7374,6 +7374,9 @@ static VALUE popen_finish(VALUE port, VALUE klass);
  *  Executes the given command +cmd+ as a subprocess
  *  whose $stdin and $stdout are connected to a new stream +io+.
  *
+ *  This method has potential security vulnerabilities if called with untrusted input;
+ *  see {Command Injection}[doc/command_injection_rdoc.html].
```

Comment:
```suggestion
 *  see {Command Injection}[command_injection.rdoc].
```

Reply:
@nobu, this change breaks the link on my local machine.  I generate the RDoc with ```rdoc --op html``` in my ruby directory.

Reply:
Sorry, I missed this reply.
It is not the way by `make html`.
You need `--page-dir doc` option.